vault/blog.2021.2021-08-30-highlights-vscode-159.md

```diff
@@ -0,0 +1,66 @@
+---
+id: JzH76BDLaFwJloUFzkGGg
+title: Highlights from VSCode 1.59
+desc: ''
+updated: 1630272832402
+created: 1630272366498
+date: '2021-08-30'
+image: "https://org-dendron-public-assets.s3.amazonaws.com/images/vscode-release-highlights-159.png"
+excerpt: "Terminal magic tricks, shared view states, and more in the VSCode 1.59 release"
+author: "Derek Ardolf"
+---
+
+> This blog article makes use of [footnotes](https://wiki.dendron.so/notes/ba97866b-889f-4ac6-86e7-bb2d97f6e376.html#footnotes), which can be useful to [academics using Dendron](https://wiki.dendron.so/notes/77c8f69c-32b8-4331-b1b2-c4211dcb9fb7.html) or for a cleaner blog experience.
+
+The open source VSCode project, along with VSCodium[^1], completed their latest monthly release with v1.59[^2]. Let's take a look at some features that the insiders build[^3] had their impatient, bleeding-edge hands on before the August 2021 release is announced.
+
+### Share view state
+
+VSCode users are often bouncing around different files in their workspaces. By default, if you close out a file tab, VSCode doesn't track where you were in that file. This can be annoying if you didn't mean to close out the file.
```

Comment:
(or because you were playing whack a mole with the new new workspace trust popups ) --> this is just me ranting against the workspace trust UX

Reply:
Snark upgrade applied